app/components/works/embargo_component.html.erb

```diff
@@ -3,7 +3,7 @@
   <div class="mb-3 row">
     <%= form.label :access, 'Who can access?', class: 'col-sm-2 col-form-label' %>
     <div class="col-sm-10">
-      <%= form.select :access, ['stanford'], {}, class: "form-select" %>
+      <%= form.select :access, [['Stanford Community', 'stanford'], ['Everyone', 'world']], { selected: "", disabled: "", prompt: "Select..." }, class: "form-select" %>
```

Comment:
Why bother with `disabled` or `selected` here?

Reply:
Without `selected` the prompt option doesn't display. Without `disabled` the prompt is a selectable option. This way includes the `Select...` prompt but it isn't a selectable option.

If there is a better patter, I'm happy to update.

Reply:
I was going by the example this rails pr https://github.com/rails/rails/pull/32087/files